src/libraries/BitmapUtils.sol

```diff
@@ -91,6 +91,25 @@ library BitmapUtils {
         return bitmap;
     }
 
+    /**
+     * @notice Converts an ordered byte array to a bitmap, validating that all bits are less than `maxSetBit`
+     * @param orderedBytesArray The array to convert to a bitmap; must be in strictly ascending order
+     * @param maxSetBit The bitmap may not contain a bit greater than or equal to this value
+     * @dev Reverts if bitmap contains a bit greater than or equal to `maxSetBit`
+     */
+    function orderedBytesArrayToBitmap(bytes memory orderedBytesArray, uint8 maxSetBit) internal pure returns (uint256) {
+        uint256 bitmap = orderedBytesArrayToBitmap(orderedBytesArray);
+
+        if (bitmap != 0) {
+            require(
+                uint8(orderedBytesArray[orderedBytesArray.length - 1]) < maxSetBit, 
+                "BitmapUtils.orderedBytesArrayToBitmap: bitmap exceeds max value"
+            );
```

Comment:
this isn't the right check
it should be checking that `bitmap < (2>> maxSetBit)`
with this check as-is, you could have a length 1 input that just set a single, too-high bit number

Reply:
I guess really should be `bitmap < (2>> (maxSetBit + 1))` if you're saying that the max bit is acceptable.
only edge case is if `maxSetBit` is 255 then

Reply:
hmm... so originally i had this mirror the validation we were doing that `bitmap <= MAX_QUORUM_BITMAP`

But then i realized what we actually wanted to check for was that the highest set bit was less than the current quorum count

Reply:
> with this check as-is, you could have a length 1 input that just set a single, too-high bit number

not sure i follow, wouldn't this be caught?

Reply:
adding here: I read this code wrong, that's my bad 🤦  This looks correct.